front/src/modules/trainschedule/components/Timetable/TimetableTrainCard.tsx

```diff
@@ -34,6 +34,9 @@ type TimetableTrainCardProps = {
   projectionPathIsUsed: boolean;
 };
 
+const formatFullDate = (d: Date) => dayjs(d).format('D/MM/YYYY HH:mm:ss');
+const formatDateHours = (d: Date) => dayjs(d).format('HH:mm:ss');
```

Comment:
It would probably be better to use the `Intl` locale instead of dayjs for formatting here, because that would then use the locale's preferred date style instead hardcoding the French format for everybody. However that'd be a change of behavior so I'd prefer to keep it in a separate patch.